src/main/java/org/breedinginsight/brapi/v2/BrAPIV2Controller.java

```diff
@@ -241,6 +251,22 @@ private HttpResponse<String> executeRequest(String path, UUID programId, HttpReq
         throw new HttpStatusException(HttpStatus.UNAUTHORIZED, "Unauthorized BrAPI Request");
     }
 
+    private HttpResponse<String> executeDeleteRequest(String path, UUID programId, HttpRequest<?> request) {
+        AuthenticatedUser actingUser = securityService.getUser();
+
```

Comment:
Maybe should add a `logCall` so behavior is consistent with executeByteRequest and executeRequest